### PR TITLE
Refactor user profile to pages

### PR DESCRIPTION
**Pull request by zacjones93**
We want to be able to link to individual sections in the user profile, the current implementation is a tab component that won't allow for this functionality.

Converted tab component to 4 pages:
- subscription
- profile
- activity
- bookmarks

These pages share a `UserLayout` that handles the navigation between pages.

All links were updated to point to `user/subscription` or `/user/profile` where previously they pointed to `/user`.

Did a little UI clean up as well removing lines that made for more noise than needed.

pages

user page walk through

Updated to separate subscription from profile page
subscription page

---
account page
activity page
bookmark page
account page mobile

### CODE REVIEW

File: src/components/pages/user/components/completed-courses.tsx [hunk 1]
```diff
@@ -51,7 +51,7 @@ const CompletedCourses: React.FC<{
               return (
                 <li
                   key={collection.slug}
-                  className="flex border-b border-gray-200 dark:border-gray-800 py-3 items-center space-x-2 pr-3 first:pt-0 last:pb-0 last:border-0"
```
Comment (zacjones93) [inline src/components/pages/user/components/completed-courses.tsx:54]: removed border on completed courses list

File: src/components/pages/user/components/widget-wrapper.tsx [hunk 2]
```diff
@@ -19,7 +19,7 @@ const ItemWrapper: React.FC<{title: string; children: React.ReactNode}> = ({
 }) => {
   return (
     <div>
-      <h2 className="pb-3 md:pb-4 text-lg font-medium md:font-normal md:text-xl leading-none border-b border-gray-200 dark:border-gray-700">
```
Comment (zacjones93) [inline src/components/pages/user/components/widget-wrapper.tsx:22]: removed border on headings

File: src/pages/user/account.tsx [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+export default Account
+
+Account.getLayout = function getLayout(Page: any, pageProps: any) {
+  return (
+    <AppLayout>
+      <UserLayout>
+        <Page {...pageProps} />
+      </UserLayout>
+    </AppLayout>
```
Comment (zacjones93) [inline src/pages/user/account.tsx:23]: User pages define a `getLayout` function that I apply `UserLayout` to

File: src/pages/user/components/user-layout.tsx [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+    path: '/user/activity',
+    name: 'Activity',
+    icon: FireIcon,
+  },
+  {
+    path: '/user/bookmarks',
+    name: 'Bookmarks',
+    icon: BookmarkIcon,
+  },
+]
```
Comment (zacjones93) [inline src/pages/user/components/user-layout.tsx:24]: defines a map of pages that will be linked to in left hand column

File: src/pages/user/components/user-layout.tsx [hunk 5]
```diff
@@ -0,0 +1,82 @@
...
+            <div className="overflow-hidden rounded-lg bg-white shadow">
+              <div className="divide-y divide-gray-200 lg:grid lg:grid-cols-12 lg:divide-y-0 lg:divide-x">
+                <aside className="py-6 lg:col-span-3">
+                  <nav className="space-y-1">
+                    {userPagesMap.map((page) => {
+                      return (
+                        <Link key={page.path} href={page.path}>
+                          <a
+                            className={cx(
+                              page.path === currentPath
```
Comment (zacjones93) [inline src/pages/user/components/user-layout.tsx:50]: the only weird thing about styling here is if a member goes to pricing page, the route will still be `/pricing` so the menu item won't be highlighted

**Review by DrShpongle — COMMENTED** [review]
lgtm 👍

**Review by DrShpongle — APPROVED** [review]

**Pull request merged**